drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/columns/EnumMultiSelectUiColumn.java

```diff
@@ -18,6 +18,7 @@
 import java.util.Arrays;
 import java.util.List;
 import java.util.Map;
+import java.util.function.Consumer;
```

Comment:
Not removed `Callback` import and one more `Callback` occurrence in this class.

Reply:
Sorry probably PR description is a bit misleading (I did it for `appformer` module and then copy/paste in all the related PRs).
The scope of the PR is to update only UberFire Wires Core Grids component with those changes and then update all the corresponding usages. It is not intended to change all `Callback` with `Consumer` in the WB codebase.